app/assets/stylesheets/application.scss

```diff
@@ -13,3 +13,5 @@
  *= require_tree .
  *= require_self
  */
+ @import "bootstrap";
```

Comment:
I don't understand why you have here changes from bootstrap branch, since on develop you don't have them yet.